src/domains/transaction/components/DelegateResignationForm/FormStep.tsx

```diff
@@ -37,12 +40,15 @@ export const FormStep = ({ fees, senderWallet }: StepProps) => {
 				<FormField name="fee" className="font-normal">
 					<FormLabel>{t("TRANSACTION.TRANSACTION_FEE")}</FormLabel>
 					<InputFee
-						value={fees.static}
-						defaultValue={fees.static}
+						value={fee}
+						defaultValue={fee}
```

Comment:
I'm not pretty sure that you need to set both `value` and `defaultValue`, since you're setting the value anyway the defaultValue will be always subscribed, no?

Reply:
We had the defaultValue required in `InputFeeProps` but better to make it optional, as it is now rewritten to use available provided values in `useFeeFormat` hook:

https://github.com/ArkEcosystem/desktop-wallet/pull/3014/files#diff-03040e39883a5a1ebe9b28e62a511f0130346de8fb9c8149fa24cfc7c29749abR11